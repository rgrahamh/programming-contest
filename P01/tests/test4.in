Blackmane Cambria 34.0000
Blackmane 136.8153 1898.6918 1617.6132 1
Ariel 139.9871 1893.4698 1611.1568 1
Alliance 142.6879 1891.1689 1612.9875 1
Cambria 148.6897 1892.4189 1605.1564 0
