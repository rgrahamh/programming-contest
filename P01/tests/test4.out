18.3172
Blackmane
Cambria
